src/components/TravelClass.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './TravelClass.css';
+
+const TravelClassDetails = ({ travelClassName, availability }) => (
+  <div className="travelClass-leftGrid">
+    <div className="travelClass-name">{travelClassName}</div>
+    <div className="travelClass-availability">{availability}</div>
+  </div>
+
+);
+const TravelClassPrice = ({ fareDetails }) => (
+  <div className="travelClass-rightGrid">
```

Comment:
can we use price component

Reply:
Price component cannot be used as it seemed to me and Nidhi that the font-size is a little bit bigger and also the height and width attached to the Price component does not sit well with the TravelClass component

Reply:
Create variants. Like how typography does or icon sizes components  do